src/com/ableton/VirtualEnv.groovy

```diff
@@ -114,16 +114,6 @@ class VirtualEnv implements Serializable {
     this.activateCommands = ". ${destDir}/${activateSubDir}/activate"
   }
 
-  /**
-   * Removes the Virtualenv from disk. You should call this method in the cleanup stage
-   * of the pipeline to avoid cluttering the build node with temporary files.
-   */
-  void cleanup() {
```

Comment:
I think we should keep this method. It gives more flexibility wrt. how the VirtualEnv objects can be used. Also, it reduces the "breakingness" of this PR.

Reply:
Sure, I can see that. See 8c3d29d.